### PR TITLE
remove the #! to python2.6 in favor of python, this was making my system barf that wants to use 2.7

### PR DESCRIPTION
**Pull request by balarsen**
- [X ] Pull request has descriptive title
- [ X] Pull request gives overview of changes
- [ n/a] New code has inline comments where necessary
- [ n/a] Any new modules, functions or classes have docstrings consistent with dbprocessing style
- [ n/a] Major new functionality has appropriate Sphinx documentation
- [ x] Added an entry to CHANGELOG if fixing a major bug or providing a major new feature
- [ n/a] New features and bug fixes should have unit tests
- [ n/a] Relevant issues are linked in the description (e.g. `See issue #` or `Closes #`)

### CODE REVIEW

**Comment by balarsen** [comment]
See #42

File: .gitignore [hunk 1]
```diff
@@ -17,3 +17,4 @@ sphinx/build/
 docs/autosummary/
 .DS_Store
 
+.cache
```
Comment (jtniehof) [inline .gitignore:20]: Any reason for this one? Seems unrelated to the rest.

**Review by jtniehof — CHANGES_REQUESTED** [review]
In theory the install should fix the shebang line based on the current interpreter, but in practice I agree this makes sense. (The 2.6 is an artifact of old scheme).

Since we have inconsistent permissions, not all scripts will be runnable without an explicit "python" on the command line, but that's another issue.

Only real query is why the gitignore update lumped in.

File: .gitignore (hunk 1 above)
Comment (balarsen) [reply .gitignore:20]: Sneaked in there from a `git status` annoyance. Can nuke or leave...

Comment (jtniehof) [reply .gitignore:20]: Think I'd rather leave it out of this PR for clarity (can you amend commit and force-push so it's still one commit?) but happy to add it in another PR if it seems generally useful, e.g. if it's something that's likely to be created on most people's systems and not just something that's happening to you.

Comment (balarsen) [reply .gitignore]: I think I did that right, please check.

Comment (jtniehof) [reply .gitignore]: Yep, looks good here.

**Review by jtniehof — APPROVED** [review]
Will leave open so @dnadeau-lanl can comment but will merge tomorrow.

**Review by dnadeau-lanl — APPROVED** [review]
Better this way! 😏

**Comment by dnadeau-lanl** [comment]
@jtniehof  Sorry I just merged!  You want me to revert!

**Comment by jtniehof** [comment]
Nope, this is good, we wouldn't be waiting for anybody else to approve :) I just didn't want to cut you off.
